src/promptflow/promptflow/_sdk/operations/_flow_operations.py

```diff
@@ -262,9 +234,13 @@ def _chat(
         """
         from promptflow._sdk._load_functions import load_flow
 
-        flow = load_flow(flow)
+        flow: ProtectedFlow = load_flow(flow)
```

Comment:
Will this only be DAG flow instance?

Reply:
Good catch, it can also be EagerFlow instance now. We should make it FlowBase here?

Reply:
I think we can.